libsrc/c64/soft80_conio.s

```diff
@@ -0,0 +1,230 @@
+;
+; Low level stuff for screen output/console input
+;
+
+        .constructor    soft80_init, 24
```

Comment:
Please add this constructor to https://github.com/cc65/wiki/wiki/Priority-of-constructors-and-destructors when this pr is merged.